apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component, PropTypes } from 'react';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
+import i18n from "@cdo/locale";
+import color from "../util/color";
+
+const styles = {
```

Comment:
These form styles are used in a few places. Does it makes sense to export them from the main form instead of duplicating them here and `SchoolNotFound`?

Reply:
:+1: That's a great idea.